src/ansible_navigator/ui_framework/ui_constants.py

```diff
@@ -0,0 +1,53 @@
+"""Constants for use with the user interface."""
+import curses
+
+from enum import IntEnum
+
+
+class Color(IntEnum):
```

Comment:
And maybe have an extra structure for `MenuColor`?

Reply:
@webknjaz It shouldn't grow at all, so I don't think we need to worry about scale here.

Reply:
For the context/history: I asked about this because I was going to send a refactoring suggestion for `color_menu()`. But I ended up deciding not to send it then so it's not relevant anymore.